coalib/collecting/Collectors.py

```diff
@@ -189,7 +189,7 @@ def filter_section_bears_by_languages(bears, languages):
     for section in bears.keys():
         new_bears[section] = tuple(
             bear for bear in bears[section]
-            if set(x.lower() for x in bear.supported_languages) & languages)
+            if {x.lower() for x in bear.LANGUAGES} & languages)
```

Comment:
IMO it'd be nice to change the variable name here. Instead of `x` use something that gives more info about the items being iterated. 
```python
for fruits in ['Apple', 'Mango']  # Good
for x in ['Apple', 'Mango']  ['Apple', 'Mango']  Bad
```